examples/typescript-reactflux/.gitignore

```diff
@@ -0,0 +1,69 @@
+
+/dist/
+/node_modules/
+/temp/
+/vendor/bower/
+/test/reports/
+/.tscache/
+/.grunt/
+/.tmp/
+/typings/*
+/src/.baseDir.ts
+!/Gruntfile.js
+!/typings/tsd.d.ts
+
+
+# Created by .gitignore support plugin (hsz.mobi)
+### JetBrains template
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm
+
+## Directory-based project format
+.idea/
+/*.iml
+# if you remove the above rule, at least ignore user-specific stuff:
+# .idea/workspace.xml
+# .idea/tasks.xml
+# .idea/dictionaries
+# and these sensitive or high-churn files:
+# .idea/dataSources.ids
+# .idea/dataSources.xml
+# .idea/sqlDataSources.xml
+# .idea/dynamic.xml
+# and, if using gradle::
+# .idea/gradle.xml
+# .idea/libraries
+
+## File-based project format
+*.ipr
+*.iws
+
+## Additional for IntelliJ
+out/
+
+# generated by mpeltonen/sbt-idea plugin
+.idea_modules/
+
+# generated by JIRA plugin
+atlassian-ide-plugin.xml
+
+# generated by Crashlytics plugin (for Android Studio and Intellij)
+com_crashlytics_export_strings.xml
+
+
+node_modules/
+build/
+public/
+
+*.sw*
+*~
+bin/
+vendor/
+*.baseDir.ts
+*.baseDir.js
+server/app.js
+
+src/**/*.js
+src/**/*.js.map
+
+.DS_Store
+*.log
```

Comment:
@arthurvr With 'global gitignore' you mean `examples/../.gitignore', right?

All the other examples come with their own `.gitignore`:

```
./ampersand/.gitignore
./angular-dart/.gitignore
./angularjs/.gitignore
./angularjs-perf/.gitignore
./angularjs_require/.gitignore
./ariatemplates/.gitignore
./atmajs/.gitignore
./backbone/.gitignore
./backbone_marionette/.gitignore
./backbone_require/.gitignore
./batman/.gitignore
./canjs/.gitignore
./canjs_require/.gitignore
./chaplin-brunch/.gitignore
./closure/.gitignore
./derby/.gitignore
./dojo/.gitignore
./duel/.gitignore
./elm/.gitignore
./emberjs/.gitignore
./enyo_backbone/.gitignore
./exoskeleton/.gitignore
./extjs_deftjs/.gitignore
./flight/.gitignore
./jquery/.gitignore
./knockback/.gitignore
./knockoutjs/.gitignore
./knockoutjs_require/.gitignore
./meteor/.meteor/.gitignore
./plastronjs/.gitignore
./polymer/bower_components/core-selector/.gitignore
./ractive/.gitignore
./rappidjs/.gitignore
./react/.gitignore
./react-backbone/.gitignore
./serenadejs/.gitignore
./socketstream/.gitignore
./typescript-angular/.gitignore
./typescript-backbone/.gitignore
./typescript-reactflux/.gitignore
./vanilladart/.gitignore
./vanillajs/.gitignore
./vue/.gitignore
./yui/.gitignore
```
Which lines should go into the global `.gitignore`?